docs/developers/protocol/hermez-protocol/protocol.md

```diff
@@ -632,16 +630,16 @@ Sender sends the transaction to a Baby Jubjub address recipient in the state tre
 If the recipient does not exist and coordinator wants to process the transaction, it should create a new account with the recipient Baby Jubjub address.
```

Comment:
sorry, I couldn't find a way to leave a comment on the correct line, under line 623, "tree" is missing an "e"

"@@ -632,16 +630,16 @@ Sender sends the transaction to a Baby Jubjub address recipient in the state tree"